docs/proposals/images/machine-states-preboot/Figure1.plantuml

```diff
@@ -0,0 +1,61 @@
+@startuml
+title Figure 1: State diagram with a generic provider
+
+(*) --> [  Machine Create  ] "Pending"
+
+note right
+- Bootstrap provider watches Machines in "pending" state,
+  generates //BootstrapConfig.Status.BootstrapData// and sets
+  //BootstrapConfig.Status.Phase// = ready.
```

Comment:
Is the idea that you'll update this in the other PR after this merges + that PR rebases?

Reply:
It should auto rebase, doesn't matter which one goes first

Reply:
I can update as part of this PR

Reply:
Done!